slm_porosity/heat_transfer.f90

```diff
@@ -39,6 +39,9 @@ MODULE user_case
   REAL (pr) :: scanning_speed
   REAL (pr) :: initial_porosity
   REAL (pr) :: initial_enthalpy
+  REAL (pr) :: dh_step              ! step for spline function maybe temporary
+  REAL (pr) :: fusion_smoother      ! half-wide of step
```

Comment:
Remove this parameter, because it is tightly connected with fixed derivative of the liquid fraction.

Reply:
Done